handlers/seed.go

```diff
@@ -0,0 +1,83 @@
+package handlers
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/layer5io/meshery/models"
+	"github.com/layer5io/meshery/models/pattern/core"
+	"github.com/layer5io/meshery/models/pattern/stages"
+	"github.com/layer5io/meshkit/utils"
+)
+
+func SeedApplications(l models.Provider) {
```

Comment:
Individually registering different types of seed content seems unnecessary. Is there value in treating content separately by type or is it more efficient to `RegisterSeedContent()`?

Reply:
I'll suggest that we're trying to be too smart here, and will only end up refactoring this logic later.

Those smarts are necessary and should be defined and executed in one location: the pattern engine (we need to give this a name).